src/ssh-generator/ssh-generator.c

```diff
@@ -246,7 +246,7 @@ static int add_vsock_socket(
                 return r;
 
         log_info("Binding SSH to AF_VSOCK vsock::22.\n"
-                 "→ connect via 'ssh vsock/%u' from host", local_cid);
+                 "Connect via 'ssh vsock/%u' from host", local_cid);
```

Comment:
use special_gylph()?

Reply:
As I wrote in the commit message, I don't think we can meaningfully check support when writing to kmsg. This may get printed on the tty, or on a serial console, and one may support utf-8 and the other not… I don't see how we can meaningfully check this.

Reply:
I changed the code to use `special_glyph()`, and I get the corrupted output again. So it detects that utf-8 is supported, but the font that is loaded does not support it.